crates/wasi-nn/examples/classification-component-onnx/Cargo.toml

```diff
@@ -2,6 +2,9 @@
 name = "classification-component-onnx"
 version = "0.1.0"
 edition = "2021"
+authors = ["The Wasmtime Project Developers"]
+readme = "README.md"
+publish = false
```

Comment:
@abrown I didn't realize that I needed to mark this as `publish = false`. Should be good now.